configs/approval.json

```diff
@@ -113,9 +131,41 @@
           "resourceDefinitions": [
             {
               "attributeFilter": {
-                "value": "*",
-                "key": "id",
-                "operation": "equal"
+                "key": "scope",
+                "operation": "equal",
+                "value": "admin"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Approval User",
+      "system": true,
+      "version": 4,
+      "description": "An approval user role which grants permissions to create/read/cancel a request, and read workflows.",
+      "access": [
+        {
+          "permission": "approval:requests:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "key": "scope",
+                "operation": "equal",
+                "value": "user"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:requests:create",
+          "resourceDefinitions": [
```

Comment:
@hsong-rh Do we need resourceDefintiion when creating a resource. I am thinking that the scope comes when we are dealing with existing records in the DB. So it would be needed for read, update, delete but not for create. In the RBAC permission check to see if a user has permission to create you would be doing
```
x = Insights::API::Common::RBAC::Access.new().process
x.accessible?('requests','create')
```

Reply:
No need to define `resourceDefinition` for regular user, only permissions are enough.